bears/r/FormatRBear.py

```diff
@@ -20,12 +20,12 @@ class FormatRBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file,
-                         r_keep_comments: bool=True,
+                         r_keep_comments: bool=None,
```

Comment:
you think we should make everything `None` and use the default style of `formatR`? I fear that the default style of formatR might change depending on version^^